src/main/java/tp04/metier/Jour.java

```diff
@@ -68,4 +75,4 @@ public String toString() {
         return "Jour{" + "annee=" + annee + ", noJour=" + noJour + '}';
     }
 
-}
+}
```

Comment:
Peux-tu ajouter un espace après l'accolade ?
```suggestion
}

```